docs/usage/configuration-options.md

```diff
@@ -2561,18 +2561,19 @@ This way Renovate can use GitHub's [Commit signing support for bots and other Gi
 
 Table with options:
 
-| Name                     | Description                                                                                                                                                |
-| ------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `bundlerConservative`    | Enable conservative mode for `bundler` (Ruby dependencies). This will only update the immediate dependency in the lockfile instead of all subdependencies. |
-| `gomodMassage`           | Enable massaging `replace` directives before calling `go` commands.                                                                                        |
-| `gomodTidy`              | Run `go mod tidy` after Go module updates. This is implicitly enabled for major module updates when `gomodUpdateImportPaths` is enabled.                   |
-| `gomodTidy1.17`          | Run `go mod tidy -compat=1.17` after Go module updates.                                                                                                    |
-| `gomodTidyE`             | Run `go mod tidy -e` after Go module updates.                                                                                                              |
-| `gomodUpdateImportPaths` | Update source import paths on major module updates, using [mod](https://github.com/marwan-at-work/mod).                                                    |
-| `npmDedupe`              | Run `npm dedupe` after `package-lock.json` updates.                                                                                                        |
-| `pnpmDedupe`             | Run `pnpm dedupe` after `pnpm-lock.yaml` updates.                                                                                                          |
-| `yarnDedupeFewer`        | Run `yarn-deduplicate --strategy fewer` after `yarn.lock` updates.                                                                                         |
-| `yarnDedupeHighest`      | Run `yarn-deduplicate --strategy highest` (`yarn dedupe --strategy highest` for Yarn >=2.2.0) after `yarn.lock` updates.                                   |
+| Name                         | Description                                                                                                                                                |
```

Comment:
By the way, the `allowedValues` part of the autogenerated table for `postUpdateOptions` does _not_ sort from A-Z. Here's the current order:

- `bundlerConservative`
- `helmUpdateSubChartArchives` <-- H comes _after_ G
- `gomodMassage`
- `gomodUpdateImportPaths`
- `gomodTidy`
- `gomodTidy1.17`
- `gomodTidyE`
- `npmDedupe`
- `pnpmDedupe`
- `yarnDedupeFewer`
- `yarnDedupeHighest`

We should probably fix the sorting, and then update the docs again to match the proper sorting? 😉

Reply:
Fixing the sorting and the docs with PR:

- #22976